packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx

```diff
@@ -101,11 +105,11 @@ export class Tabs {
     );
   }
 
-  private hasIcon(icon) {
+  private hasIcon(icon: string) {
     return icon && (icon as string) !== '';
   }
```

Comment:
Not a full review. but I thought I'd comment on this as it caught my eye:

- I'd probably do `icon?: string` as it is optional
- An empty string is a falsy value, so the last part of this check will never be false

So, unless I am mistaken by some curiosity of Stencil or web components, I would have done this:

```javascript
private hasIcon(icon?: string) {
  return Boolean(icon);
}
```

Reply:
Ah. That makes sense. thankyou!